src/biom.cpp

```diff
@@ -309,7 +427,12 @@ unsigned int biom::get_sample_data_direct(const std::string &id, uint32_t *& cur
 }
 
 double* biom::get_sample_counts() {
-    double *sample_counts = (double*)calloc(sizeof(double), n_samples);
```

Comment:
Why not use directly calloc?
It is shorter and probably faster.

Reply:
Good point, left over from when attempting to parallelize this